src/test/java/sorinfoca/Tests/Tema8/TestPage/TestPages.java

```diff
@@ -0,0 +1,56 @@
+package sorinfoca.Tests.Tema8.TestPage;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import sorinfoca.Tests.Tema8.pages.*;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class TestPages {
+    private static WebDriver driver;
+    private static WebDriverWait wait;
+
+    @Test(groups = "desktop")
```

Comment:
I don't see any test that can be run on mobile group, I see only desktop for tests. You can double check if same scenarios will work for mobile too and if not please add according scenarios for mobile. Either way, don't forget to add mobile group too. Thanks!